packages/git/src/browser/blame/blame-decorator.ts

```diff
@@ -20,6 +20,9 @@ import { GitFileBlame } from '../../common';
 import { Disposable, DisposableCollection } from '@theia/core';
 import * as moment from 'moment';
 import URI from '@theia/core/lib/common/uri';
+import { DecorationStyle } from '@theia/core/lib/browser';
+
+const blameDecorationsStyleSheet: CSSStyleSheet | undefined = DecorationStyle.createStyleSheet();
```

Comment:
Here as well, please make this a field on the class so that it can be accessed by extenders.